content/en/serverless/step_functions/troubleshooting.md

```diff
@@ -41,6 +41,15 @@ Please enable the `Include execution data` option on the state machine's logging
 - For actions, we support basic actions of Lambda and DynamoDB. For example, Lambda Invoke, DynamoDB GetItem, DynamoDB PutItem, DynamoDB UpdateItem and more.
 - `Wait`, `Choice`, `Success`, `Fail`, and `Pass` are supported, while `Map` and `Parallel` are not. You are able to see parallel executing spans stacked on top of each other, but no `Parallel` spans show on the flame graph.
 
+## Customized way to deploy Datadog Lambda Forwarder
+If you are using your customized way to deploy Datadog Lambda Forwarder, here are some tips that can help you debug enabling Step Functions tracing:
+- On the forwarder, please set the environment variable `DD_FETCH_STEP_FUNCTIONS_TAGS` to `true`. 
+- Datadog-Forwarder layer version greater than 31 should be able to fetch state machine tags, which is a requirement for fetching `DD_TRACE_ENABLE` tag on state machines to enable Step Functions trace generation on Datadog backend.
```

Comment:
I'm not sure what this means. Are you telling the user they need to have a Datadog-Forwarder layer version greater than 31?

```suggestion
- To to enable Step Functions trace generation on the Datadog backend, the Datadog-Forwarder layer version must be greater than 31. The Datadog-Forwarder fetches state machine tags, which is a requirement for fetching `DD_TRACE_ENABLE` tag on state machines.
```

Reply:
Yes, and the reason is that we need to get this particular tag which feature only comes after layer version 31 I believe.

Reply:
@estherk15 ^ thank you!

Reply:
@kimi-p How about this?
```suggestion
- To enable Step Functions trace generation on the Datadog backend, the Datadog-Forwarder layer version must be greater than 31. This version is able to fetch state machine tags, which is a requirement for fetching the`DD_TRACE_ENABLE` tag on state machines.
```

Reply:
Does this make sense to you? @estherk15 
```
- To enable Step Functions trace generation on the Datadog backend, the Datadog-Forwarder layer version must be greater than 31. This version is able to fetch state machine tags, including the required `DD_TRACE_ENABLED` tag.
```

Reply:
Nice! Yes, that's much clearer
```suggestion
- To enable Step Functions trace generation on the Datadog backend, the Datadog-Forwarder layer version must be greater than 31. This version is able to fetch state machine tags, including the required `DD_TRACE_ENABLED` tag.
```

Reply:
Awesome, thank you!

Reply:
Please help me merge once it's build.